.github/workflows/create-new-image.yml

```diff
@@ -0,0 +1,19 @@
+name: create-new-image
```

Comment:
I don't think this workflow creates the image. From the looks of it it just creates a branch that's then used by push-new-image workflow to actually build and push the image. I think the name of this workflow should be adjusted to avoid misunderstandings.

Reply:
Fixed in https://github.com/newrelic/newrelic-php-daemon-docker/pull/75/commits/2a3dd698a8fb2812527de7c2ed2a912361ba3a55